src/beta/realtime/websocket.ts

```diff
@@ -26,6 +26,7 @@ export class OpenAIRealtimeWebSocket extends OpenAIRealtimeEmitter {
     props: {
       model: string;
       dangerouslyAllowBrowser?: boolean;
+      onUrl?: (url: URL) => void;
```

Comment:
nit
```suggestion
      onURL?: (url: URL) => void;
```

also would be nice to mention / mark this as internal-only
```suggestion
      /**
       * Callback to mutate the URL, needed for Azure.
       * @internal
       */
      onURL?: (url: URL) => void;
```

Reply:
Addressed in https://github.com/openai/openai-node/pull/1287/commits/9a73f6f3043644e3d2879a3e5284fe75273db45a